trie1.py

```diff
@@ -0,0 +1,43 @@
+#Time Complexity: O(length)
+#Space Complexity: O(length)
```

Comment:
Space occupied by the Tries DS is O(n*L), where n is the number words in the tree and L is the average length of all the words.